apps-rendering/src/components/Byline/GalleryByline.tsx

```diff
@@ -1,24 +1,39 @@
 import type { SerializedStyles } from '@emotion/react';
 import { css } from '@emotion/react';
-import { text } from '@guardian/common-rendering/src/editorialPalette';
+import { border, text } from '@guardian/common-rendering/src/editorialPalette';
 import type { ArticleFormat } from '@guardian/libs';
-import { remSpace } from '@guardian/source-foundations';
+import { headline, remSpace } from '@guardian/source-foundations';
 import type { Option } from '@guardian/types';
 import { grid } from 'grid/grid';
 import { darkModeCss } from 'styles';
-import { defaultAnchorStyles, DefaultByline } from './Byline.defaults';
+import { DefaultByline } from './Byline.defaults';
 
 const bylineStyles = (format: ArticleFormat): SerializedStyles => css`
 	padding-top: ${remSpace[2]};
 	padding-bottom: ${remSpace[2]};
-	${grid.span('centre-column-start', 3)}
+	${grid.column.centre}
 	grid-row: 6/7;
 	color: ${text.gallery(format)};
+	font-style: italic;
+
 	${darkModeCss`
 		color: ${text.galleryDark(format)};
 	`};
 `;
 
+const anchorStyles = (format: ArticleFormat): SerializedStyles => css`
+	${headline.xxxsmall({ fontWeight: 'bold' })}
```

Comment:
I think the bits in between the links are falling back to a body font? This should put everything in an italic "Guardian Headline", and a mix of medium and bold:

```suggestion
	${headline.xxxsmall({ fontWeight: 'medium', fontStyle: 'italic' })}

	${darkModeCss`
		color: ${text.galleryDark(format)};
	`};
`;

const anchorStyles = (format: ArticleFormat): SerializedStyles => css`
	${headline.xxxsmall({ fontWeight: 'bold', fontStyle: 'italic' })}
```

Reply:
All true about the commas etc!